src/backend/api/reservations.js

```diff
@@ -0,0 +1,78 @@
+const express = require("express");
+const router = express.Router();
+const knex = require("../database");
+
+//select all
+router.get("/", async (request, response) => {
+  try {
+    const reservations = await knex("reservation").select("*");
+    response.json(reservations);
+  } catch (error) {
+    throw error;
+  }
+});
+module.exports = router;
+//add new reservation
+router.post("/", async (request, response) => {
+  try {
+    const addReservation = request.body;
+    const [reservationId] = await knex("reservation").insert({
+      number_of_guests: addReservation.number_of_guests || "Untitled",
```

Comment:
`number_of_guests` should be a number sot he "Untitled" fallback value does not seem right. Perhaps 0 or 1 instead?